server/src/main/java/org/apache/iotdb/db/engine/flush/FlushManager.java

```diff
@@ -93,6 +95,10 @@ public void runMayThrow() {
             tsFileProcessor.getTsFileResource().getTsFile().getAbsolutePath());
       }
       registerTsFileProcessor(tsFileProcessor);
+      // update stat monitor cache to system during each flush()
+      if (IoTDBDescriptor.getInstance().getConfig().isEnableStatMonitor()) {
+        StatMonitor.getInstance().cacheStatValue();
```

Comment:
It is better to just flush the flushed SG.

Reply:
Fixed.